docs/api/logstash-configuration-management/create.asciidoc

```diff
@@ -0,0 +1,50 @@
+[role="xpack"]
+[[logstash-configuration-management-api-create]]
+=== Create Pipeline
+
+experimental[This functionality is experimental and may be changed or removed completely in a future release.]
```

Comment:
nit: The saved object api has the word `experimental` in this warning in bold to help call attention to it at a glance. My thought process was that when an api graduated to beta or stable, it would be more obvious to a developer that frequently referenced the docs for it if the status itself was bold.